### PR TITLE
chore: pin pandas and ray versions for ray tests

### PR DESCRIPTION
**Pull request by hamidzr**
https://github.com/search?q=repo%3Aray-project%2Fray+pandas+language%3AText&type=code&l=Text (cherry picked from commit 6d1a5967f6e85ec15d5473c0a15f677cb8a1a1f0)

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by hamidzr** [comment]
waiting on this run to finish https://app.circleci.com/pipelines/github/determined-ai/determined?branch=hz-new-ami

File: examples/features/ports/startup-hook.sh [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
 pip install 'pydantic<2.0.0' # resolves 'pydantic.dataclasses.dataclass only supports init=False'
-pip install -U "ray[air]"
+pip install -U "ray[air]==2.9.3"
+pip install 'pandas==1.5.3'
+pip show pandas
+pip show pydantic
+pip show ray
```
Comment (ioga) [inline examples/features/ports/startup-hook.sh:8]:
```suggestion
```

**Review by ioga — APPROVED** [review]
lgtm, thanks for taking care of this

File: examples/features/ports/startup-hook.sh (hunk 1 above)
Comment (hamidzr) [reply examples/features/ports/startup-hook.sh]: ops that triggered another ci cycle. automerge ftw

**Comment by hamidzr** [comment]
thanks for the help

**Pull request merged**